pkg/config/sampling/sampling.go

```diff
@@ -77,10 +81,33 @@ func (u *Config) Get() *corev1.ConfigMap {
 	}
 }
 
+// CheckForSamplingConfigFile will check if there is a config file present
+// if there is one it returns true
+func CheckForSamplingConfigFile(jaeger *v1.Jaeger) bool {
+	jsonObject, err := jaeger.Spec.Sampling.Options.MarshalJSON()
```

Comment:
The info at `jaeger.Spec.Sampling.Options` is used to populate the sampling strategies file, when configured via the CR. The `sampling.strategies-file` option would be defined in `jaeger.Spec.AllInOne.Options` or `jaeger.Spec.Collector.Options`, depending on whether the `jaeger.Spec.Strategy` value, e.g.
```
if jaeger.Spec.Strategy == DeploymentStrategyAllInOne {
  options = jaeger.Spec.AllInOne.Options
} else {
  options = jaeger.Spec.Collector.Options
}
```

Reply:
fixed in recent commit